modules/sdk-core/src/bitgo/environments.ts

```diff
@@ -194,7 +194,7 @@ const mainnetBase: EnvironmentTemplate = {
   cronosNodeUrl: 'https://cronos-pos-rest.publicnode.com/',
   initiaNodeUrl: 'https://rest.initia.xyz/',
   asiNodeUrl: 'https://rest-fetchhub.fetch.ai/',
-  dotNodeUrls: ['wss://rpc.polkadot.io'],
+  dotNodeUrls: ['wss://asset-hub-polkadot-rpc.n.dwellir.com'],
```

Comment:
Where did you get this link? Seems to differ from https://www.dwellir.com/networks/asset-hub